### PR TITLE
freedv: remove unused private fields

### PR DESCRIPTION
**Pull request by marcusmueller**
This makes compilation a little more silent.

### CODE REVIEW

File: gr-vocoder/python/vocoder/qa_freedv.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/ usr / bin / env python
```
Comment (mbr0wn) [inline gr-vocoder/python/vocoder/qa_freedv.py:1]: Both the shebang and the copyright year seem wrong. The shebang can go, right? This doesn't get manually executed, IIRC.

File: gr-vocoder/python/vocoder/qa_freedv.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        dec = vocoder.freedv_rx_ss()
+        self.assertTrue(enc)
+        self.assertTrue(dec)
+
+    def test002_allthemodes(self):
+        for idx, (mode, name) in enumerate(modes.items()):
+            interleaves = (-1, )
+            if mode == freedv_api.MODE_700D:
+                interleaves = (1, 4, 16, -1, 0)
+            #print(f"Mode {name} ({mode})")
```
Comment (mbr0wn) [inline gr-vocoder/python/vocoder/qa_freedv.py:44]: leftover debug?

File: gr-vocoder/python/vocoder/qa_freedv.py (hunk 1 above)
Comment (marcusmueller) [reply gr-vocoder/python/vocoder/qa_freedv.py:1]: well, it's usual that our QA codes have that, IIRC; I'll check (I personally regularly call qa_foobar_stuff.sh manually, but that calls python3 qa_foobar_stuff.py)

File: gr-vocoder/python/vocoder/qa_freedv.py (hunk 2 above)
Comment (marcusmueller) [reply gr-vocoder/python/vocoder/qa_freedv.py:44]: yea

File: gr-vocoder/python/vocoder/qa_freedv.py (hunk 1 above)
Comment (marcusmueller) [reply gr-vocoder/python/vocoder/qa_freedv.py:1]: Don't know how that happened, though, and that kinda worries me

**Comment by marcusmueller** [comment]
@mbr0wn fixed the copyright and leftover commented out debug print; the shebang is left for consistency reasons

**Comment by mbr0wn** [comment]
@marcusmueller The FreeDV QA failed on F29.

**Comment by marcusmueller** [comment]
@mbr0wn ... turns out that this is the only CI worker that has Codec2 / FreeDV even available.
Yeah, fails for me now, too. FreeDV seems to have a bug in mode 700D that seems to overwrite libc malloc tables/headers, so that a `free` call in `freedv_close` spectacularly goes wrong.

I tried circumventing that by simply disabling that `freedv_close` in the dtor if your mode is 700D, but then the python interpreter crashes later on when it collects memory, so something's pretty broken.

Since I can't be debugging FreeDV right now: Ways forward?

* Unconditionally disable MODE_700D?
* Disable it, but have an "enable_known_bad" gr::prefs?
* ???

**Comment by mbr0wn** [comment]
> Unconditionally disable MODE_700D?

Let's do it and wait until someone complains.

**Comment by dkozel** [comment]
Maitland is the maintainer for Codec2/FreeDV on Debian and I think in active contact with the devs. Lets file an issue here about FreeDV and get the Debian and Ubuntu CI builders testing FreeDV/Codec2 then ask him to take a look?

**Comment by mbr0wn** [comment]
@marcusmueller lack of activity, please reopen if you get the issues fixed.

**Pull request closed without merging**